File: examples/python/otp_generate.py
import os
import sys

from kalourmade_arkesel import ArkeselClient

number = sys.argv[1] if len(sys.argv) > 1 else sys.exit("Usage: python otp_generate.py <number>")
client = ArkeselClient(os.environ["ARKESEL_API_KEY"])

result = client.otp.generate(
    expiry=5, length=6, medium="sms",
    message="Your Arkesel Python SDK test code is %otp_code%",
    number=number, sender_id="Arkesel", type="numeric",
)

print(f"Generated: {result.message}")

File: examples/node/otpGenerate.ts
import { ArkeselClient } from '@kalourmade/arkesel-sms';

const number = process.argv[2];
if (!number) {
  console.error('Usage: tsx otpGenerate.ts <number>');
  process.exit(1);
}

const client = new ArkeselClient(process.env.ARKESEL_API_KEY!);
const result = await client.otp.generate({
  expiry: 5,
  length: 6,
  medium: 'sms',
  message: 'Your Arkesel Node SDK test code is %otp_code%',
  number,
  senderId: 'Arkesel',
  type: 'numeric',
});

console.log(`Generated: ${result.message}`);
